bundles/org.openhab.binding.solax/src/main/resources/OH-INF/thing/channel_types.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="solax"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="frequency">
+		<item-type>Number</item-type>
+		<label>Electric frequency</label>
+		<description>Electric Frequency</description>
+	</channel-type>
+	<channel-type id="battery-temperature">
+		<item-type>Number:Temperature</item-type>
+		<label>Battery Temperature</label>
+		<description>Battery Temperature</description>
+	</channel-type>
+	<channel-type id="lastRetrieveTimeStamp">
+		<item-type>DateTime</item-type>
+		<label>Last retrieve time stamp</label>
+		<description>Last time with a successful retrieval of data</description>
+	</channel-type>
+	<channel-type id="rawDataType">
```

Comment:
What is the benefit of making a channel advanced? Do you have to explicitly enable it somehow?

Reply:
It will be hidden and you need to set a checkmark to show advanced channels. See also https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types